### PR TITLE
Added talawa mobile offline first features documentation

### LINKED ISSUE #2548: Document all the Offline First Features developed so far as part of GSoC.
**Is your feature request related to a problem? Please describe.**
Currently the offline first features developed as part of GSoC project for the mobile application lack an extensive documentation support on talawa docs.

**Describe the solution you'd like**
Create markdown files explaining all the models, services, flows, features in the mobile app and add them to [talawa-docs](https://docs.talawa.io/)

**Describe alternatives you've considered**

**Approach to be followed (optional)**
A clear and concise description of approach to be followed.

**Additional context**

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

### PR DESCRIPTION
**Pull request by Azad99-9**
**What kind of change does this PR introduce?**

**Issue Number:**

Fixes [2548](https://github.com/PalisadoesFoundation/talawa/issues/2548) of talawa mobile

**Did you add tests for your changes?**

**Snapshots/Videos:**

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-docs/blob/develop/CONTRIBUTING.md)?**

### CODE REVIEW

**Comment by Azad99-9** [comment]
@palisadoes 

I would also like to mention [this](https://www.figma.com/board/ARjXjQej896ox1fvH9yR4s/Untitled?node-id=25-891&node-type=text&t=gI3MZfBrjWiCbrZy-0) figma link as part of documentation how can I do it.

**Comment by palisadoes** [comment]
> @palisadoes
> 
> I would also like to mention [this](https://www.figma.com/board/ARjXjQej896ox1fvH9yR4s/Untitled?node-id=25-891&node-type=text&t=gI3MZfBrjWiCbrZy-0) figma link as part of documentation how can I do it.

@aashimawadhwa has access to our official Figma documents and will be able to access to update them.

The links can be found here:

- https://docs.talawa.io/docs/design/ux/ux-design-system

File: package.json [hunk 1]
```diff
@@ -25,7 +25,8 @@
 		"prettier": "^3.3.3",
 		"prism-react-renderer": "^2.4.0",
 		"react": "^18.2.0",
-		"react-dom": "^18.2.0"
+		"react-dom": "^18.2.0",
+		"yarn": "^1.22.22"
```
Comment (Azad99-9) [inline package.json:29]: @palisadoes I have run `npm yarn install` in development so this file has been changed please have a look. Resulting in an edit of unauthorised file.

**Comment by aashimawadhwaa** [comment]
@Azad99-9  as this is a figma jam file we can create a separate reference to it from the talwa main website. keeping it in the figma is not a great idea @palisadoes

**Comment by palisadoes** [comment]
@aashimawadhwaa Should we revert this?

**Comment by Azad99-9** [comment]
@aashimawadhwaa could you please tell me how exactly should I proceed?

**Pull request merged**